nameko/cli/run.py

```diff
@@ -168,7 +169,14 @@ def main(args):
             import_service(path)
         )
 
-    config = {AMQP_URI_CONFIG_KEY: args.broker}
+    config = {}
+
+    if args.config:
+        with open(args.config) as fle:
+            config = yaml.load(fle)
+    else:
+        config[AMQP_URI_CONFIG_KEY] = args.broker
```

Comment:
Do we want to keep ``--broker`` as an argument? What about adding ``--web-server-address``?

Obviously requiring every possible config value to be specified as a command line option is ridiculous, but maybe allowing a subset would be helpful. And then which subset?

Is there a world where the runner allows any possible config to be specified on the command line, and it's up to the user to use a config file when the invocation gets silly? It would require us to start registering config keys, but that's something I'd like to do anyway -- each Extension could specify the config it requires and default values; the ServiceContainer could do the same (for ``MAX_WORKERS`` and ``PARENT_CALLS_TRACKED``).